pom.xml

```diff
@@ -51,6 +51,8 @@
     <copy-rename-maven-plugin>1.0.1</copy-rename-maven-plugin>
     <maven-release-plugin.version>3.0.0-M7</maven-release-plugin.version>
     <maven-surefire-plugin.version>3.0.0-M7</maven-surefire-plugin.version>
+    <jacoco-maven-plugin.version>0.8.7</jacoco-maven-plugin.version>
```

Comment:
if this wasn't already here, let's not add it.  to run jacoco without it in here you can do the long form, e.g. `mvn test org.jacoco.jacoco-maven-plugin:report` (i think)